bindings/python/eblob_python.cpp

```diff
@@ -52,6 +53,29 @@ static void eblob_extract_id(const struct eblob_id &e, struct eblob_key &id)
 	eblob_extract_arr(e.id, id.id, &len);
 }
 
+/**
+ * __eblob_read_ll() - interruption-safe wrapper for pread(2)
+ */
+static int __eblob_read_ll(int fd, char *data, size_t size, off_t offset)
```

Comment:
>what will prevent client code from running forever ignoring signals, since it returns back into syscall right after it has ignored signal?

Client code may be a server side with specific signal handling.